src/main/resources/default/taglib/t/singleSelect.html.pasta

```diff
@@ -51,6 +52,9 @@
 </div>
 <script>
     sirius.ready(function () {
+        const _dependencyField = document.getElementById('@dependencyField');
+        const _dependencyFieldSelect = document.getElementById('@dependencyField' + '-select');
```

Comment:
**Required:** If the other field is also a TokenAutocomplete the [val](https://github.com/scireum/token-autocomplete/blob/master/lib/token-autocomplete.ts#L288C5-L288C5) function should be used instead of relying on the internal structure of the component. This could change in the future and break this functionality.